### PR TITLE
Upload volume tasks

### LINKED ISSUE #4401: Allow creating multiple volume tasks from zipfile
A zipfile (format see volume project download) uploaded to task creation should create one new task per tracing with that tracing as initial data

### PR DESCRIPTION
**Pull request by youri-k**
### URL of deployed dev instance (used for testing):
- https://uploadvolumetasks.webknossos.xyz

### Steps to test:
- Upload a single zip containing multiple volume zips (e.g. project download)
- Should create as many tasks as included volume zips

### Issues:
- fixes #4401 

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- [ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes
- [ ] Needs datastore update after deployment
- [x] Ready for review

### CODE REVIEW

**Comment by youri-k** [comment]
I just noticed that hybrid tasks aren't fully supported, yet. I'll do this today...

**Comment by youri-k** [comment]
Update is done. I tested it locally. Maybe you could do another check and after that I think, it's good to go live.

**Comment by fm3** [comment]
I’ll have another look today :) let’s wait until monday with merging, though

File: app/controllers/TaskController.scala [hunk 1]
```diff
@@ -235,46 +236,103 @@ class TaskController @Inject()(annotationDAO: AnnotationDAO,
...
+                              fullParams: List[TaskParameters],
+                              taskType: TaskType,
+                              organizationId: ObjectId)(
+      implicit ctx: DBAccessContext,
+      m: MessagesProvider): Fox[(List[Option[SkeletonTracing]], List[Option[(VolumeTracing, Option[File])]])] =
+    if (taskType.tracingType == TracingType.skeleton)
+      if (volumeBaseOpts.forall(_.isEmpty)) Fox.failure(Messages("taskType.mismatch", "skeleton", "volume"))
+      else Fox.successful((skeletonBaseOpts, volumeBaseOpts))
+    else if (taskType.tracingType == TracingType.volume)
+      if (skeletonBaseOpts.forall(_.isEmpty)) Fox.failure(Messages("taskType.mismatch", "volume", "skeleton"))
```
Comment (fm3) [inline app/controllers/TaskController.scala:255]: this does not work for me as expected. could it be that this is the wrong way round? what exactly is expected for each tasktype?

Comment (youri-k) [reply app/controllers/TaskController.scala]: Yes, you are correct. I just changed it. Maybe you could have another look :slightly_smiling_face:

**Comment by youri-k** [comment]
Yeah, I'll open an issue

**Pull request merged**